melodic/lcas-dist.yaml

```diff
@@ -555,10 +555,47 @@ repositories:
     status: developed
   spencer_people_tracking:
     source:
-      test_commits: true
+      test_commits: false
       type: git
-      url: https://github.com/LCAS/spencer_people_tracking.git
-      version: master
+      url: https://github.com/spencer-project/spencer_people_tracking.git
+      version: melodic
+    release:
+      packages:
+      - pcl_people_detector
+      - rwth_ground_hog
+      - rwth_ground_plane
+      - rwth_perception_people_msgs
+      - rwth_upper_body_detector
+      - spencer_bagfile_tools
+      - spencer_control_msgs
+      - spencer_detected_person_association
+      - spencer_detected_person_conversion
+      - spencer_diagnostics
+      - spencer_group_tracking
+      - spencer_human_attribute_msgs
+      - spencer_leg_detector_wrapper
+      - spencer_people_tracking_full
+      - spencer_people_tracking_launch
+      - spencer_perception_mocks
+      - spencer_social_relation_msgs
+      - spencer_social_relations
+      - spencer_tracking_metrics
+      - spencer_tracking_msgs
+      - spencer_tracking_rviz_plugin
+      - spencer_tracking_utils
+      - spencer_vision_msgs
+      - srl_laser_detectors
+      - srl_laser_features
+      - srl_laser_segmentation
+      - srl_nearest_neighbor_tracker
+      - srl_tracking_exporter
+      - srl_tracking_logfile_import
+      - track_annotation_tool
+      - video_to_bagfile
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/lcas-releases/spencer_people_tracking.git
```

Comment:
As said, this shouldn't be here, but I have reconfigured the release repository in https://github.com/lcas-releases/spencer_people_tracking/commit/fa68d4a2dfa5348f322bb135357ead1acccb7d65 so that a new release is pulled from your fork and the correct branch.

Reply:
Ok great!